tests/checkpointing/test_model_checkpoint.py

```diff
@@ -14,27 +14,39 @@
 import os
 import os.path as osp
 from unittest import mock
-
 import pytorch_lightning as pl
-from unittest.mock import Mock
-
-import yaml
 import pickle
 import platform
 import re
+from distutils.version import LooseVersion
 from pathlib import Path
+from unittest.mock import MagicMock, Mock
 
 import cloudpickle
 import pytest
 import torch
+import yaml
 
+import pytorch_lightning as pl
 import tests.base.develop_utils as tutils
 from pytorch_lightning import Trainer, seed_everything
 from pytorch_lightning.callbacks import ModelCheckpoint
 from pytorch_lightning.loggers import TensorBoardLogger
-from tests.base import EvalModelTemplate, BoringModel
 from pytorch_lightning.utilities.cloud_io import load as pl_load
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from tests.base import BoringModel
+
+
+class LessBoringModel(BoringModel):
```

Comment:
as this is not inside a test function, can we have more description name?

Reply:
Done